README.md

```diff
@@ -1,28 +1,37 @@
 # PSScriptAnalyzer Action
-Github action for running PSScriptAnalyzer and use ConvertToSARIF to generate a SARIF file.
 
-# Getting Started
-To run this action add the step below in your GitHub Action:
+Github action for running PSScriptAnalyzer and use ConvertToSARIF to generate a Static Analysis
+Results Interchange Format (SARIF) file. For more information about SARIF, see the
+[SARIF website][16].
+
+## Getting Started
+
+To run this action, add the following step to your GitHub Action:
+
 ```yaml
- - name: Run PSScriptAnalyzer
-   uses: microsoft/psscriptanalyzer-action@v1.0
-   with:
+- name: Run PSScriptAnalyzer
+  uses: microsoft/psscriptanalyzer-action@v1.0
+  with:
     path: .\
-    recurse: true 
+    recurse: true
     output: results.sarif
 ```
-The above yaml code scans all the code in your repository and outputs the results to `result.sarif` at the CWD.
 
-# YAML
-See the [input section](#Inputs) for more info about the inputs.
+This YAML code scans all  code in your repository and writes the results to `result.sarif`
```

Comment:
Superfluous whitespace between `all` and `code` here. Also saying "This action" instead of "This YAML code" would be more idiomatic and less redundant.